### ISSUE TITLE
Test suite link is dead?

### ISSUE THREAD

**Issue by jrlanglois** [dbry/WavPack#98]
http://www.rarewares.org/wavpack/test_suite.zip does not appear to work anymore... Which is unfortunate seeing that I was hoping to re-test the decoder I have at hand.

**Comment by dbry** [comment]
I just downloaded the suite from there. Perhaps it was just down temporarily?

I notice that on Chrome I have to right-click to get it. If I just left-click it does nothing, which might be because that's not a secure link and secure links do work with left-click. Security feature?

**Comment by jrlanglois** [comment]
Looks like you're right! I do feel a bit silly now, though Chrome didn't prompt _anything_ whatsoever when left-clicking the link.

**Comment by dbry** [comment]
Yeah, it's weird that it just does nothing rather than some sort of pop-up.

I found a setting that changes the behavior. It's under `chrome://flags` and it's called `Treat risky downloads over insecure connections as active mixed content`. If you change that to `disabled` it will work correctly (although it's probably not a good idea).
